<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оценка случайности</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>270</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Алгоритмические значения</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Табличные значения</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="alg_table">
   <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>301</width>
      <height>281</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>0..9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>10..99</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>100..999</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table_table">
   <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>301</width>
      <height>281</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>0..9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>10..99</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>100..999</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Мера случайности</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>340</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Мера случайности</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_alg_1">
   <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_alg_2">
   <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_alg_3">
   <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_table_3">
   <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_table_1">
   <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_table_2">
   <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fill_alg">
   <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Заполнить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fill_table">
   <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Заполнить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>421</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Ручной ввод:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="manual_input">
   <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>451</y>
      <width>661</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>471</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Мера случайности:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_manual">
   <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>475</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
